modtools/create-unit.lua

```diff
@@ -566,6 +587,7 @@ if args.name then
 else
   local unit = df.unit.find(unitId)
   unit.name.has_name = false
+  unit.name.first_name = "" -- remove the first name altogether. gets rid of units having number names.
```

Comment:
Hold on I went out of town after loading 1.25, this is from my phone... okay the issue was that when you view them either with v or k I think on the map it shows up, additionally i thought it was in the unit list... I can understand the arena mode thing... We could add a verify of arena mode and ignore that line?

I won't be able to test any code until at least Monday evening,  it's not a dire fix more of a 'visual immersion fix.  It was primarily being used her for summoned/purchased pets, a user side fix is easy enough for that.

Another note: I'm not sure but github appears to have waited 3 days to tell me you wrote here? Lol